lib/block_transpose.in.cu

```diff
@@ -57,10 +60,19 @@ namespace quda
 
       template <typename vAccessor, typename bAccessor> void launch_device_(TuneParam &tp, const qudaStream_t &stream)
       {
-        Arg<true, vAccessor, bAccessor> arg(V, B, tp.block.x, tp.block.y);
-        tp.set_max_shared_bytes = true;
-        resizeVector(tp.block.y * tp.grid.y); // We need a full threadblock
-        launch_device<BlockTransposeKernel>(tp, stream, arg);
+        if (from_to_non_rel) {
```

Comment:
Should this be `if constexpr`?

Reply:
`from_to_non_rel` is not a template argument here, but the line right after this can add a `constexpr`.

Reply:
https://github.com/lattice/quda/pull/1497/commits/87d332f5178d97dc15e642d362f1e5d4e1f2bfd8